### PR TITLE
Get rid of reload(fronend) calls in tests

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #480

### CODE REVIEW

File: ga4gh/serverconfig.py [hunk 1]
```diff
@@ -84,3 +86,29 @@ class TestOidcConfig(TestConfig):
     OIDC_AUTHZ_ENDPOINT = "https://accounts.example.com/auth"
     OIDC_TOKEN_ENDPOINT = "https://accounts.example.com/token"
     OIDC_TOKEN_REV_ENDPOINT = "https://accounts.example.com/revoke"
+
+
+class FlaskDefaultConfig(object):
```
Comment (jeromekelleher) [inline ga4gh/serverconfig.py]: This is a bit ugly --- is there any other way to reset to a Flask default? If we keep this we should add some comments explaining that it's only used in testing.

Comment (dcolligan) [inline ga4gh/serverconfig.py]: I don't think so.  "Modifying" flask for us usually means modifying the config, which is just, essentially, a dictionary.  Reseting means getting this dictionary back to the defaults.  The Flask API does not seem to provide any method to do this for us: http://flask.pocoo.org/docs/0.10/api/#flask.Config

Comment (jeromekelleher) [inline ga4gh/serverconfig.py]: OK, the comments at least make it clear that this is what we're doing. Flask configuration really isn't all that amazing...

**Comment by jeromekelleher** [comment]
+1

**Pull request merged**